src/ChainSafe.GamingSDK.EVM/Web3/Core/Evm/Dto/Chains.cs

```diff
@@ -49,7 +49,7 @@ private static void CaptureError(string error, Web3Environment environment)
                 { "error", error },
             };
 
-            environment.AnalyticsClient.CaptureEvent("Chains", properties);
+            // todo log error using new event format
```

Comment:
Out of curiosity will this `todo` work in IDE warning system you need proper `TODO:`?

Reply:
It works in VS. No idea about Rider though.

Reply:
Yeah, it will in Rider. All `todo` no matter the case and `throw new NotImplementedException();` show up in Rider's todo tab.